packages/backend/Dockerfile

```diff
@@ -19,15 +19,17 @@ RUN apt-get update && \
     yarn config set python /usr/bin/python3
 
 # From here on we use the least-privileged `node` user to run the backend.
-USER node
 WORKDIR /app
+RUN chown node:node /app
```

Comment:
without this I got:

```
 => ERROR [ 7/10] RUN tar xzf skeleton.tar.gz && rm skeleton.tar.gz                                                                                                                                                                                                      0.2s
------
 > [ 7/10] RUN tar xzf skeleton.tar.gz && rm skeleton.tar.gz:
#11 0.155 tar: packages: Cannot mkdir: Permission denied
#11 0.155 tar: packages/backend/package.json: Cannot open: No such file or directory
#11 0.155 tar: packages: Cannot mkdir: Permission denied
#11 0.155 tar: packages/app/package.json: Cannot open: No such file or directory
#11 0.155 tar: Exiting with failure status due to previous errors
```

Reply:
Is that with the reordering of the user command? I think we should be fine as long as you declare the user before the workdir?

Reply:
Yes, it is with the reordering, the untar wasn’t working for me without this

Reply:
seems to, see https://github.com/backstage/backstage/pull/13747